lib/hexagram/repositories/repository.rb

```diff
@@ -11,6 +11,10 @@ def find(id)
         map_record_out(orm_adapter.find(id, persistence_class))
       end
 
+      def all()
```

Comment:
If a method doesn't take parameters, convention is to leave off the parenthesis https://github.com/bbatsov/ruby-style-guide#method-parens